src/overlays/actors/ovl_En_Bsb/z_en_bsb.c

```diff
@@ -124,123 +204,1681 @@ ActorInit En_Bsb_InitVars = {
+
+    if (this->unk_02B0 == OBJECT_BSB_LIMB_NONE) {
+        Audio_RestorePrevBgm();
+        Collider_DestroyJntSph(play, &this->collider);
+    }
+    if (CutsceneManager_GetCurrentCsId() == this->csIdList[3]) {
+        CutsceneManager_Stop(this->csIdList[3]);
+    }
+}
+
+s32 func_80C0B888(EnBsb* this, PlayState* play) {
+    Player* player = GET_PLAYER(play);
+    s16 yaw;
+
+    if (!this->unk_02DC) {
+        yaw = ABS_ALT((s16)(this->actor.yawTowardsPlayer - this->unk_02BC));
+        if (yaw < (s16)(BREG(1) + 0x4000) && (fabsf(this->actor.world.pos.y - player->actor.world.pos.y) < 20.0f) &&
+            (this->actor.xzDistToPlayer < 200.0f)) {
+            this->unk_02DC = true;
+            Audio_PlayBgm_StorePrevBgm(NA_BGM_MINI_BOSS);
+            return true;
+        }
+    } else {
+        return true;
+    }
+
+    return false;
+}
+
+void func_80C0B970(EnBsb* this, PlayState* play) {
+    s16 quakeY;
+
+    if (Animation_OnFrame(&this->skelAnime, 8.0f) || Animation_OnFrame(&this->skelAnime, 14.0f)) {
+        Actor_PlaySfx(&this->actor, NA_SE_EN_KTIA_WALK);
+
+        quakeY = (700.0f - this->actor.xzDistToPlayer) * 0.01f;
+
+        if (quakeY > 4) {
+            quakeY = 4;
+        }
+
+        if (quakeY > 0) {
+            Actor_RequestQuakeAndRumble(&this->actor, play, quakeY, 2);
+        }
+
+        if (Animation_OnFrame(&this->skelAnime, 8.0f)) {
+            func_80C0B31C(play, this, &this->unk_0304);
+        } else {
+            func_80C0B31C(play, this, &this->unk_02F8);
+        }
+    }
+}
+
+void func_80C0BA58(EnBsb* this, PlayState* play) {
+    s16 sourceIntensity;
+
+    if (Animation_OnFrame(&this->skelAnime, 2.0f) || Animation_OnFrame(&this->skelAnime, 4.0f) ||
+        Animation_OnFrame(&this->skelAnime, 6.0f) || Animation_OnFrame(&this->skelAnime, 8.0f)) {
+        Actor_PlaySfx(&this->actor, NA_SE_EN_KTIA_WALK);
+        Actor_RequestQuake(play, BREG(10) + 1, BREG(11) + 2);
+        if (Animation_OnFrame(&this->skelAnime, 4.0f) || Animation_OnFrame(&this->skelAnime, 8.0f)) {
+            func_80C0B31C(play, this, &this->unk_0304);
+        } else {
+            func_80C0B31C(play, this, &this->unk_02F8);
+        }
+    }
+
+    if (this->actor.xzDistToPlayer < BREG(7) + 400.0f) {
+        sourceIntensity = BREG(8) + 160.0f - this->actor.xzDistToPlayer * 0.3f;
+
+        if (sourceIntensity > 70) {
+            sourceIntensity = 70;
+        } else if (sourceIntensity < 0) {
+            sourceIntensity = 0;
+        }
+
+        if (Animation_OnFrame(&this->skelAnime, 1.0f)) {
+            Rumble_Override(BREG(15), BREG(12) + sourceIntensity, BREG(13) + 15, BREG(14) + 70);
+        }
+    }
+}
+
+s32 func_80C0BC30(EnBsb* this) {
+    if ((this->actor.world.pos.z < -2180.0f) && (this->actor.world.pos.z > -2470.0f) &&
+        (fabsf(this->actor.world.pos.y - this->actor.home.pos.y) < 30.0f)) {
+        Vec3s sp38 = { 0x2EE0, -0x2710, 0xB5C8 };
+        Vec3s sp30 = { -0x2710, 0x2710, -0x2710 };
```

Comment:
```suggestion
        Vec3s sp38 = { 0x2EE0, -0x2710, -0x4A38 };
        Vec3s sp30 = { -0x2710, 0x2710, -0x2710 };
```
I think this makes more sense?